hardhat/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
```

Comment:
```suggestion

npx hardhat compile
```

It seems that hardhat needs to be compiled before running `run.sh`.

Reply:
done